src/cli/cmds/init.ts

```diff
@@ -146,6 +116,16 @@ class ModuleVersion {
   public get mavenDependency() {
     return this.mavenVersion;
   }
+
+  private resolveVersion(module: string): string {
+    if (module === 'cdk8s-cli') {
+      module = '../../../';
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-require-imports
+    return require(`${module}/package.json`).version;
```

Comment:
I'm not sure I follow this. Why would `cdk8s-core` and `cdk8s-plus-17` be installed at this point?

Reply:
They are needed by the init templates so the way the CLI decides which version to take is by taking a dependency on them (previously we already had a dep on `cdk8s` and now I've added `cdk8s-plus-17`). I think it's a good way to ensure that the CLI works with these versions because now we have tests in the CLI that verify that init templates work.

Naturally, after a project is created, users can upgrade to a newer version if they want.

Reply:
Gotcha. Sounds good.